dualscreeninfo/android/src/main/kotlin/com/microsoft/reactnativedualscreen/dualscreen/DualScreenInfo.kt

```diff
@@ -67,31 +67,49 @@ class DualScreenInfo constructor(context: ReactApplicationContext) : ReactContex
 			return displayMetrics.heightPixels - (rectangle.top + rectangle.height());
 		}
 
+	private val mNavigationBarWidth: Int
+		get() {
+			val rectangle = Rect()
+			val displayMetrics = DisplayMetrics()
+			currentActivity?.window?.decorView?.getWindowVisibleDisplayFrame(rectangle)
+			currentActivity?.windowManager?.defaultDisplay?.getRealMetrics(displayMetrics);
+			return displayMetrics.widthPixels - (rectangle.left + rectangle.width());
+		}
+
 	private val windowRects: List<Rect>
 		get() {
 			val boundings = mDisplayMask?.getBoundingRectsForRotation(rotation)
 			val barHeights = mStatusBarHeight + mNavigationBarHeight
 			val windowBounds = windowRect;
 			return if (boundings == null || boundings.size == 0) {
-				windowBounds.bottom = windowBounds.bottom - barHeights;
+				if (rotationToOrientationString(mRotation) == "portrait" || rotationToOrientationString(mRotation) == "portraitFlipped") {
+					//single screen portrait
+					windowBounds.bottom = windowBounds.bottom - barHeights
+				} else {
+					//single screen landscape
+					windowBounds.right = windowBounds.right - mNavigationBarWidth
```

Comment:
if you need to adjust for the side nav bar, shouldn't this be subtracting its height and not width?

Reply:
In landscape it's a right inset, so yes we want to subtract the width.

Reply:
Just so we're clear on terms here - I define the bar width, as its longer dimension along the screen edge, whereas its height is its 'thickness', ie. how much it stick out into the middle of the screen from the edge

what is mNavigationbarWidth exactly here?

Reply:
Ah okay, so: height of the nav bar == width of the right inset in landscape.

Reply:
Ok, then follow-up question: is the thickness of the bottom and side nav bar the same? If so, then I think you should rename this variable